projects/widgets/docs/getting-started.md

```diff
@@ -1,3 +1,66 @@
-# getting started
+# Getting started
 
-...
+## 1. Registering a widget project
+
+Before you can start building your widget you must make a widget project on our [self-service platform Projectaanvraag](https://projectaanvraag.uitdatabank.be). The registration process is very simple and only takes a few minutes.
+
+The created widget project will use event data from our test environment. No payment is required until you decide you want to use real event data.
+
+<https://www.youtube.com/watch?v=X0uplBw98gg>
+
+<!-- theme: success -->
+
+> Once you have created your widget project, you're all set to start building your widget!
+
+## 2. Building your widget
+
+### Choosing a template 🗂️
+
+To help you get started quickly, we offer 4 different templates:
+
+1. **Leeg**: You start from a blanc layout that you then expand yourself by adding rows and assigning blocks (e.g. search box or search results) to these rows.
+2. **Agenda**: This is the most used template. This templates contains two rows. The top row contains a a search box (full length), and the second row contains facets on the left s ide and search results at the center.
+3. **Tips**: This template can be used to make a short list of events that you want to highlight. The tips widget serves as a teaser for the full event calendar. The tips widget is often used to generate traffic to the full calendar. Therefore it is usually embedded on the home page or another frequently visited page.
+4. **UiTPAS**: This is the same template as the `agenda` template, but with all the UiTPAS setting already enabled for you.
+
+Regardless of the template you pick, you can later on extensively customize your widget to your own preferences and taste:
+
+* by changing the layout: adding, moving or deleting rows and blocks
+* by configuring the different blocks: determine which events and information you want to show in your agenda
+
+👉 [Learn more about widget templates](./templates.md)
+
+### Configuring your widget 🏗️
+
+Now that you've chosen a template it's time to define:
+
+* which events you want to display in your agenda
+* which info you want to show about the events
+
+The way of configuring is very simple and is mainly done by ticking and unchecking checkboxes.
+
+👉 [Learn more about the  configuration settings](./configuration.md)
+
+### Publishing and embedding your widget 💾
+
+Once you're satisfied with the configuration you've done in step 2 you can publish your widget. The publish action will save all configuration changes you've made on the `<script>` tag of your widget.
```

Comment:
The "save all configuration changes you've made on the `<script>` tag of your widget" might make it seem like all configuration is stored inside the `<script>` tag (which you copy & paste on your website), and you will need to update the `<script>` tag on your website when you update the configuration? At least that's how I could see it being interpreted but it's subjective of course.

Reply:
You could of course just mention somewhere (for example after the next sentence below) that you only need to embed the script tag once, and that you don't need to change it after every configuration update